client/src/types/index.ts

```diff
@@ -10,6 +11,59 @@ export type Asset = {
   organization: number;
 };
 
+export type Organization = {
+  id?: number;
+  name: string;
+  doing_business_as: string;
+  description: string;
+  website: string;
+  address: string;
+  phone: string;
+  city: string;
+  state: string;
+  ein: string;
+  nonprofit_classification: string;
+};
+
+export type UserOrg = {
+  id?: number;
+  approvalStatus: ApprovalStatus;
+  role: Role;
+};
```

Comment:
🙃 i'm removing userOrg type in my PR
lmk if this effects your work here. if so we can look at conflicts together and see what best route is

Reply:
on second thought, let's try to get yours landed and then i can do mine after
but still with this entity, it should be created under the hood i.e. FE doesn't know anything about it
pretty sure this will be possible if we do the 1 form pattern i mentioned

Reply:
Thanks, I moved everything into 1 form